assets/component-rating.css

```diff
@@ -14,6 +14,7 @@
 }
 
 .rating-star {
+  --color-rating-star: rgb(var(--color-foreground));
```

Comment:
I could see the loss of accent color for rating stars being one of the more noticeable cases. So I've at least captured the color in a variable up at the .rating-star scope so would be easy to change via custom css. Even though the color gets used in a gradient style within a pseudo element.

`.rating-star { --color-rating-star: #000; }`